src/components/redditClone/posts.js

```diff
@@ -1,12 +1,13 @@
 import React from 'react'
+import f from 'styles/frontpage'
 
 export default props => {
```

Comment:
this file is called posts (plural) but it displays a single post. We should rename it everywhere.